pkg/datapath/linux/node.go

```diff
@@ -51,6 +53,18 @@ const (
 	failed       = "failed"
 )
 
+const (
+	neighListFileName = "neigh-links.json"
+)
+
+// NeighLinks is a slice of NeighLinks
+type NeighLinks []NeighLink
```

Comment:
There is going to be only one node used for neighDiscovery. What's a reason for having the slice?

Reply:
That's true today but we don't know if that will also be true tomorrow.

Reply:
Yeah, then let's add the multi link support tomorrow, but not today. I don't get why we should increase complexity of the code base for features which most likely won't be even implemented.